sidechain/consensus/aura/src/verifier.rs

```diff
@@ -81,13 +81,8 @@ where
 			self.slot_duration,
 		);
 
-		verify_author::<AuthorityPair, ParentchainBlock::Header, SignedSidechainBlock, _>(
-			&slot,
-			signed_block.block(),
-			parentchain_header,
-			ctx,
-		)?;
-
+		// We check the ancestry before the author, because the parentchain block might not match
+		// in case we try to import a sidechain block that was already imported longer than 1 parentchain block ago.
```

Comment:
This comment is not quite self-explanatory if one does not know the background of it.

Suggestion:
`We need to check the ancestry first to ensure that an already imported block does not result in an author verification error.`

Reply:
Yes, you're right. I'll update the comment 👍